storage/objects/objects_test.go

```diff
@@ -422,6 +425,9 @@ func TestPostPolicyV4(t *testing.T) {
 	bucketName := tc.ProjectID + "-post-policy-bucket-name"
 	objectName := "foo.txt"
 	serviceAccount := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
+	if serviceAccount == "" {
```

Comment:
I'm kind of dubious about this actually-- if something changes in how we do creds the test will be silently skipped. @codyoss thoughts?

Reply:
I think this is fine, but I don't think it is needed since this will always be set in the case of system tests. https://github.com/GoogleCloudPlatform/golang-samples/blob/60e4b3700f89663d46ddd80fdca3a70c6326fa7d/testing/kokoro/system_tests.sh#L106

If this changed I think a lot would break.

Reply:
I think the check is useful for local testing/debugging. I changed it to output an error rather than skip the test. Is that okay or should I just remove the check? Note that the `TestV4SignedURL()` in this file also does this check and skips if not set.

Reply:
Either way is fine with me.